### PR TITLE
11.0 mig analytic department

### PR DESCRIPTION
**Pull request by Tonow-c2c**

### CODE REVIEW

File: analytic_department/LICENSE
Comment (elicoidal) [inline analytic_department/LICENSE]: remove this and merge the relevant content in README

File: analytic_department/README.rst
Comment (elicoidal) [inline analytic_department/README.rst]: svg ->png

File: analytic_department/README.rst
Comment (elicoidal) [inline analytic_department/README.rst]: update branch

File: analytic_department/__manifest__.py
Comment (elicoidal) [inline analytic_department/__manifest__.py]: target repo URL

File: analytic_department/i18n/analytic_department.pot
Comment (elicoidal) [inline analytic_department/i18n/analytic_department.pot]: remove this file

File: analytic_department/views/analytic.xml
Comment (elicoidal) [inline analytic_department/views/analytic.xml]: improve the format for readibility

**Review by elicoidal — CHANGES_REQUESTED** [review]
mostly details.
LGTM

**Comment by feketemihai** [comment]
@Tonow-c2c @elicoidal I don't see the reason of having a new module just to add some views that should be in the dependant module, so i suggest to improve analytic_base_department, because there we add the fields.

**Comment by elicoidal** [comment]
@feketemihai what you suggest makes sense actually!

**Comment by damdam-s** [comment]
@feketemihai @elicoidal we did this way because this module exists in v9 so we migrated this one. 

BTW, if other people agree, no problems to improve the `analytic_base_department` module.

@gurneyalex @pedrobaeza any opinion?

**Comment by pedrobaeza** [comment]
Yes, I see the merge reasonable if you don't mind.

**Comment by feketemihai** [comment]
@Tonow-c2c Can you please squash last 2 commits, otherwise :+1:

**Comment by Tonow-c2c** [comment]
@elicoidal it seems better to you ?

File: analytic_base_department/__manifest__.py
Comment (elicoidal) [inline analytic_base_department/__manifest__.py:11]: Are you sure you need to explicit the dependency?

File: analytic_base_department/views/analytic.xml
Comment (elicoidal) [inline analytic_base_department/views/analytic.xml]: Could you improve the formating for readability?
Are you sure the icon is available in newest Odoo versions?
(a `grep terp-folder-orange . -r` in standard v11 returns nothing)

**Review by elicoidal — CHANGES_REQUESTED** [review]

File: analytic_base_department/__manifest__.py
Comment (damdam-s) [reply analytic_base_department/__manifest__.py:11]: @elicoidal I think explicit is better than implicit :smile: 

Of course not needed but it doesn't bother IMHO

Comment (damdam-s) [reply analytic_base_department/__manifest__.py:11]: @elicoidal you really want @Tonow-c2c to delete the explicit dependency or we can go with it?

Comment (elicoidal) [reply analytic_base_department/__manifest__.py:11]: not blocking for me @damdam-s @Tonow-c2c

**Comment by damdam-s** [comment]
@feketemihai @pedrobaeza can you please review and eventually approve :smile: ?
@elicoidal thanks for not blocking this one. Can you please "dismiss" your review or make another one in order to remove "red and requested changes"?

Have a nice day

**Review by elicoidal — APPROVED** [review]
(no functional tests)

**Comment by feketemihai** [comment]
@damdam-s I will still need the squash of last commits...

**Comment by Tonow-c2c** [comment]
@feketemihai squash done :wink:

**Review by tschanzt — APPROVED** [review]
Tested. works and no issue with the changes.

**Review by p-tombez — APPROVED** [review]
LGTM, code review only

**Review by damdam-s — APPROVED** [review]
thanks

File: analytic_base_department/README.rst
Comment (simahawk) [inline analytic_base_department/README.rst:18]: update whole readme according to https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst

**Review by simahawk — CHANGES_REQUESTED** [review]

**Comment by feketemihai** [comment]
@Tonow-c2c can you squash last commits, and i think this should be merged...

@pedrobaeza

**Comment by Tonow-c2c** [comment]
@feketemihai Commit squashed :wink:

**Review by simahawk — APPROVED** [review]

**Pull request merged**
